packages/frontend-api/src/Model/Resolver/Settings/SeoSettingsQuery.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrontendApiBundle\Model\Resolver\Settings;
+
+use Shopsys\FrameworkBundle\Component\Domain\Domain;
+use Shopsys\FrameworkBundle\Model\Seo\SeoSettingFacade;
+use Shopsys\FrontendApiBundle\Model\Resolver\AbstractQuery;
+
+class SeoSettingsQuery extends AbstractQuery
+{
+    /**
+     * @param \Shopsys\FrameworkBundle\Component\Domain\Domain $domain
+     * @param \Shopsys\FrameworkBundle\Model\Seo\SeoSettingFacade $seoSettingFacade
+     */
+    public function __construct(
+        protected readonly Domain $domain,
+        protected readonly SeoSettingFacade $seoSettingFacade,
+    ) {
+    }
+
+    /**
+     * @return array{robots: string[]}
+     */
+    public function seoSettingsQuery(): array
+    {
+        $robotsContent = $this->seoSettingFacade->getRobotsContent($this->domain->getId());
+
+        return [
+            'robots' => $robotsContent !== null ? preg_split('/\r\n|[\r\n]/', $robotsContent) : [],
```

Comment:
This transformation should be done using transformers so whenever this setting is loaded, it is in "final" format and when it is displayed in the form it is in form format.

Reply:
Also a question - why do we need to return this as an array instead of a string?

Reply:
I originally wanted to do this to unify line breaks but it's true that it's probably not necessary. I modified it back to string.